opp_gui/include/opp_gui/widgets/database_selection_widget.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2019 Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OPP_GUI_WIDGETS_DATABASE_SELECTION_WIDGET_H
+#define OPP_GUI_WIDGETS_DATABASE_SELECTION_WIDGET_H
+
+#include <geometry_msgs/Transform.h>
+
+#include <opp_gui/widgets/database_browser_widget.h>
+
+namespace opp_application
+{
+class ApplicationContextBase;
+typedef typename std::shared_ptr<ApplicationContextBase> ApplicationContextBasePtr;
+typedef typename boost::function<void(bool)> NotifyCbType;
```

Comment:
This alias isn't being reference in this header file as far as I can tell

Reply:
It was being referenced in the source file.  I removed the header and source, as this widget was not in use.